notebooks/slicing_tutorial_WiC.ipynb

```diff
@@ -0,0 +1,864 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snorkel-Workshop: Slicing Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "To start, let's make sure that we have the right paths/environment variables set by following the instructions in `snorkel-superglue/README.md`.\n",
+    "\n",
+    "Specifically, ensure that:\n",
+    "* `snorkel` is installed\n",
+    "* `PYTHONPATH` includes root of `snorkel-superglue`\n",
+    "* `SUPERGLUEDATA` is set where [download_superglue_data.py](https://github.com/HazyResearch/snorkel-superglue/blob/staging/download_superglue_data.py) was called."
```

Comment:
The actual execution of these commands is down below in the next chunk; move them up here.